awscli/customizations/s3/subcommands.py

```diff
@@ -512,7 +512,7 @@ def _list_all_objects(self, bucket, key, page_size=None,
             self._display_page(response_data)
 
     def _display_page(self, response_data, use_basename=True,
-                      strip_prefix=None):
+                      key=''):
```

Comment:
Would it be possible to rename this to something like `provided_prefix` or `user_prefix`? Using the name key is not necessarily correct as that value was used as the `Prefix` value to list objects v2 and it is a little ambiguous on where the key value came from, the response or user input.